app/main.py

```diff
@@ -1,34 +1,72 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self, start: tuple, end: tuple, is_drowned: bool = False
+                 ) -> None:
+        self.ship = []
+        if start == end:
+            self.ship.extend([Deck(*start)])
+        elif start[1] != end[1]:
+            self.ship.extend([Deck(start[0], deck)
+                              for deck in range(start[1], end[1] + 1)])
+        elif start[0] != end[0]:
+            self.ship.extend([Deck(deck, start[1])
+                              for deck in range(start[0], end[0] + 1)])
```

Comment:
Use for loop instead of checks

Reply:
It will decrease the speed of function

Reply:
why do you think so?

Reply:
try to write and check the speed of work

Reply:
I can't remove check's because they are checking is ship going to vertical or horizontal